gui/wxpython/datacatalog/tree.py

```diff
@@ -1355,6 +1353,7 @@ def OnPasteMap(self, event):
                         self._removeMapAfterCopy(
                             self.copy_layer[i], self.copy_mapset[i], env2
                         )
+                        self.showNotification.emit(message=_("Moving completed"))
```

Comment:
Messages make sense, maybe just a small code suggestion - I would put    `else: self.showNotification.emit(message=_("Copying completed"))` below this and remove rows 1341-1344? 
We could also make the condition positive (instead of if not self.copy_mode:, use if self.copy_mode?)

Reply:
+1 for some more shuffling here. `if self.copy_mode:` followed by `if not self.copy_mode:` is hard to understand.

Reply:
You mean like in 
f2c7eff 
?

Reply:
Great :-)